src/Cake.Cli/Cake.Cli.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>Cake.Cli</AssemblyName>
+    <TargetFrameworks>netstandard2.0;net5.0</TargetFrameworks>
```

Comment:
Since this is also used in Cake.Core it should also target `net46`

Reply:
It's not used in `Cake.Core`, it references `Cake.Core` though.

For `net46` won't work because of dependencies, it could and probably should target `net461` too as `Cake.exe` does.

I can add that.